AssetTracker/apps/importcsv/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from ..assets.models import Asset
+
+class AssetForm(forms.ModelForm):
+  class Meta:
+    model = Asset
+    fields = ("asset_tag", "device_type", "device_model", "asset_status", "serial_number", "asset_condition", "screen_size", "hard_drive", "ram", "year", "created_by", "company")
```

Comment:
Maybe the use of single and double quotes, but not quite sure. We can chat about it on the reetro

Reply:
Let's leave doubles for now, I look up what is the convention and it's hard to figure out actually